include/UI/UIView.h

```diff
@@ -220,16 +255,45 @@ class UIView
 
   vec2f m_histogram_point_1;
   vec2f m_histogram_point_2;
-  float m_histogram_quantiles[2]; 
+  float m_histogram_quantiles[2];
+
 
-  
   void load_ocean_color_maps();
 
   std::vector<std::string> m_ocean_color_maps_names;
 
   std::string m_color_map_directory;
 
+  float m_animation_speed;
+  std::string m_str_animation_duration;
```

Comment:
why is the animation duration a string?

Reply:
I renamed the variable to m_animation_duration_to_string. This is to display the float number on the ui text.

Reply:
Why not make it a function that returns a string then? Do you keep `m_animation_duration` for any other reason than to display it on the UI?

Reply:
Yes. It's just a place holder for the  UI for the imgui::inputtextfield.